RadixWallet/Features/AssetsFeature/Components/NonFungibleAssetList/Components/Details/NonFungibleTokenDetails+View.swift

```diff
@@ -170,6 +182,16 @@ extension NonFungibleTokenDetails {
 	}
 }
 
+extension OnLedgerEntitiesClient.StakeClaim {
+	var description: String {
+		if isReadyToBeClaimed {
+			L10n.AssetDetails.Staking.readyToClaim(claimAmount.formatted())
+		} else {
+			L10n.AssetDetails.Staking.unstaking(claimAmount.formatted(), reamainingEpochsUntilClaim * 5)
```

Comment:
what is `5` here?

Reply:
I meant to extract as a constant, it is the Epoch duration.